os/fs/driver/mtd/smart.c

```diff
@@ -5036,10 +5036,25 @@ static int smart_journal_release_sector(FAR struct smart_struct_s *dev, uint16_t
 	offset += offsetof(struct smart_sect_header_s, status);
 	ret = MTD_WRITE(dev->mtd, offset, 1, &header.status);
 	if (ret != 1) {
-		fdbg("Release Invalid data sector Error ret : %d target sector : %d\n", ret, psector);
+		fdbg("Release sector Error ret : %d target sector : %d\n", ret, psector);
 		return -EIO;
 	}
 	
+	/* TODO Now verify release here but it just 1 byte, I guess fail never happened as per my experience...
```

Comment:
Why the need for re-verification of status bytes after marking it as released?

Reply:
Powercut test environment takes 4~5 second until it power down totally, during this time it may process file operation in very low working condition.
So we should check this one byte has written properly or not.